taverna-cwl-activity-ui/src/main/java/org/apache/taverna/cwl/ui/serviceprovider/Testing.java

```diff
@@ -18,141 +18,63 @@
 
 import java.io.File;
 import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.FilenameFilter;
-import java.util.ArrayList;
-import java.util.HashMap;
+import java.io.IOException;
+import java.io.Writer;
+import java.util.Arrays;
+import java.util.Iterator;
 import java.util.LinkedHashMap;
 import java.util.Map;
 
-import org.apache.taverna.cwl.CwlActivityConfigurationBean;
+import org.apache.jena.atlas.iterator.Iter;
+import org.openjena.atlas.json.JsonObject;
+import org.springframework.util.SystemPropertyUtils;
 import org.yaml.snakeyaml.Yaml;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
 public class Testing {
```

Comment:
If this class is for testing, can we move it to same package, but under `src/test/java` ? 

I'm not sure what it is testing.. is it more for experimenting?  If it can be rewritten to a junit test with `@Test` methods, that would be ideal.